fmriprep/workflows/anatomical.py

```diff
@@ -47,13 +67,50 @@ def t1w_preprocessing(name='t1w_preprocessing', settings=None):  # pylint: disab
     flt_wmseg_sbref = pe.Node(fsl.FLIRT(dof=6, bins=640, cost_func='mutualinfo'),
                               name="WMSeg_2_SBRef_Brain_Affine_Transform")
 
+    invert_wmseg_sbref = pe.Node(
+        fsl.ConvertXFM(invert_xfm = True),
+        name="invert_wmseg_sbref"
+    )
+
+    t1_2_mni = pe.Node(ants.Registration(), name = "T1_2_MNI_Registration")
+    t1_2_mni.inputs.fixed_image = op.join(get_mni_template(), 'MNI152_T1_2mm.nii.gz')
+    t1_2_mni.inputs.metric = ['Mattes'] * 3 + [['Mattes', 'CC']]
```

Comment:
fdf9e06 and 	b852031 fix this. I am re running the workflow to ensure the values are being pulled, this will take a few hours.